mp/src/game/shared/momentum/run/mom_replay_data.h

```diff
@@ -3,6 +3,11 @@
 #include <momentum/util/serialization.h>
 #include "utlbuffer.h"
 
+
+// HACK: To keep compatibility, store teleport flag in the buttons
+// Remember to update me if more button flags are added!!!
+#define IN_REPLAY_TELEPORTED            (1 << 27)
```

Comment:
I wouldn't worry about maintaining compatibility; we plan on modifying the replay file format in the near future anyway.

Reply:
From what I understood, a refactoring is happening on the replay code at some point, so I didn't want to add anything new. I think this can be done once refactoring is done.